transmonee_dashboard/src/transmonee_dashboard/pages/__init__.py

```diff
@@ -521,16 +631,30 @@ def geocode_address(address):
 col_types = {
     "COVERAGE_TIME": str,
     "OBS_FOOTNOTE": str,
+    "OBS_VALUE": str,
     "Frequency": str,
     "Unit multiplier": str,
 }
 
+# api parameters: query format and year limits
+api_params = {
+    "format": "csv",
+    "startPeriod": str(years[0]),
+    "endPeriod": str(years[-1]),
+}
+compress_header = {"Accept-Encoding": "gzip"}
+
 # avoid a loop to query SDMX
 try:
-    sdmx = pd.read_csv(sdmx_url.format("+".join(inds)), dtype=col_types)
+    response = requests.get(
+        sdmx_url.format("+".join(inds)), params=api_params, headers=compress_header
+    )
```

Comment:
why are we making a requests call here when `pd.read_csv` will do this already?

Reply:
I think I see why mostly this is for inclusion of the headers, pandas can already accommodate this: 

`sdmx_url = f"https://sdmx.data.unicef.org/ws/public/sdmxapi/rest/data/ECARO,TRANSMONEE,1.0/.{'+'.join(inds)}....?format=csv&startPeriod={years[0]}&endPeriod={years[-1]}"`

`pd.read_csv(sdmx_url, dtype=col_types, storage_options={"Accept-Encoding": "gzip"})`

This is documented here: https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html

Reply:
@beto-Sibileau take a look at this

Reply:
@jamescw thank you James, well noted, haven't thought about it before... I'll take a look at the documentation, maybe we could add the parameters format, startPeriod, endPeriod as well, would make the url more clear (not very relevant, just getting picky =)

Reply:
> @beto-Sibileau take a look at this

@james any clue on why this is not being considered as a HTTPS url?

```
sdmx_url = "https://sdmx.data.unicef.org/ws/public/sdmxapi/rest/data/ECARO,TRANSMONEE,1.0/.{}....?format=csv&startPeriod={}&endPeriod={}"
sdmx = pd.read_csv(
    sdmx_url.format("+".join(inds), years[0], years[-1]),
    dtype=col_types,
    storage_options={"Accept-Encoding": "gzip"},
    )
```

The error I got looks to me related to the fact that the URL is being handled by fsspec

`ValueError: storage_options passed with file object or non-fsspec file path`


Pandas doc:

![image](https://user-images.githubusercontent.com/62593695/125078777-2b9ebd00-e0c3-11eb-99e2-016dd9318eb7.png)

Reply:
LGTM, only issue to add is the headers